frontend/src/pages/PipelineList.tsx

```diff
@@ -176,6 +177,8 @@ class PipelineList extends Page<{}, PipelineListState> {
         request.pageSize,
         request.sortBy,
         request.filter,
+        this.props.namespace ? 'NAMESPACE' : undefined,
```

Comment:
How do people view shared pipelines?
Shall we add a checkbox that allows showing shared pipelines instead?
What about pipeline selector list?

Reply:
Right now the query pulls the pipelines within the namespace, and the public pipelines. We could add the ability to query either shared pipeline or the pipelines for the namespace only.

Reply:
I see, I think it'll save some future engineering efforts if we first discuss on the desired behavior in https://github.com/kubeflow/pipelines/issues/4197 and reach a consensus among main contributors.